content/kubernetes/re-clusters/delete_custom_resources.md

```diff
@@ -11,66 +11,72 @@ aliases: [
+kubectl delete -f operator.yaml
+```
 
-The operator attaches a finalizer to the Redis Enterprise cluster (REC) object. This makes sure the Redis cluster is deleted before the REC custom resource is removed from the K8s cluster.
+You will also need to remove [the `namespaceSelector` section from the validating webhook]({{relref "/kubernetes/deployment/quick-start#webhook"}}).
+
+### Troubleshoot REDB deletion
+
+The operator attaches a finalizer to the Redis Enterprise database (REDB) object. This makes sure the database is deleted before the REDB custom resource is removed from the K8s cluster.
 
-If the finalizer isn't removed automatically by the operator, you won't be able to delete your REC resource.
+If the operator isn't running, or some other fatal error occurs, the finalizer isn't removed automatically by the operator. In this case, you won't be able to delete your REDB resource.
 
 If this happens, you can remove the finalizer manually.
 
 {{<warning>}} If you remove the finalizer manually, there is not guarantee that the underlying REC has been deleted. This may cause resource issues and require manual intervention. {{</warning>}}
 
 ```sh
-kubectl patch rec <your-rec-name> --type=json -p \
-    '[{"op":"remove","path":"/metadata/finalizers","value":"redbfinalizer.redisenterpriseclusters.app.redislabs.com"}]'
+kubectl patch redb <your-db-name> --type=json -p \
+    '[{"op":"remove","path":"/metadata/finalizers","value":"finalizer.redisenterprisedatabases.app.redislabs.com"}]'
 ```
 
-## Delete the operator
+### Troubleshoot REC deletion
 
-To delete the operator from your K8s cluster and namespace, you can delete the operator bundle with `kubectl delete -f bundle.yaml`. This will remove the operator and its custom resource definitions (CRDs) from your K8s cluster.
+The operator attaches a finalizer to the Redis Enterprise cluster (REC) object. This makes sure the Redis cluster is deleted before the REC custom resource is removed from the K8s cluster.
 
-{{<warning>}} The Redis Enterprise CRDs are non-namespaced resources, meaning they are shared across your entire K8s cluster. Deleting CRDs in one namespace will delete custom resources in every other namespace across the K8s cluster.{{</warning>}}
+If the operator isn't running, or some other fatal error occurs, the finalizer isn't removed automatically by the operator. In this case, you won't be able to delete your REDB resource.
 
-If you have Redis Enterprise clusters running in different namespaces on the same K8s cluster, deleting the entire operator bundle might cause data loss.
+If this happens, you can remove the finalizer manually.
 
-To safely delete the operator from one namespace without affecting the others, delete the operator files individually, excluding the CRD files:
+{{<warning>}} If you remove the finalizer manually, there is not guarantee that the underlying REC has been deleted. This may cause resource issues and require manual intervention. {{</warning>}}
```

Comment:
```suggestion
{{<warning>}} If you remove the finalizer manually, there is no guarantee that the underlying REC has been deleted. This may cause resource issues and require manual intervention. {{</warning>}}
```